packages/cli/src/services/checkly-config-loader.ts

```diff
@@ -79,10 +79,12 @@ function isString (obj: any) {
   return (Object.prototype.toString.call(obj) === '[object String]')
 }
 
-export async function loadChecklyConfig (dir: string, filenames = ['checkly.config.ts', 'checkly.config.js']): Promise<{ config: ChecklyConfig, constructs: Construct[] }> {
+export async function loadChecklyConfig (dir: string, filename = ''):
```

Comment:
```suggestion
export async function loadChecklyConfig (dir: string, filename?: string):
```

`filename = ''` also works, so feel free to leave as-is. I personally prefer `filename?: string`, though, since we want the `filename` parameter to be optional.